src/Duke.java

```diff
@@ -0,0 +1,225 @@
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Duke {
+
+    private static int Count = 0;
+    private static Task[] List = new Task[10]; //creating list of Task for inputs
```

Comment:
Maybe could try to avoid the use of magic number. Can maybe set it to a named constant